style.css

```diff
@@ -0,0 +1,25 @@
+
+.container{
+    margin: 0 auto;
+    max-width: 1325px;    
+}
+
+
+header {
+    display: flex;
+    justify-content: space-between;
+    padding: 30px 0px 0px 0px;
+    margin: 0px -40px 0px -40px;
```

Comment:
Why do you need to use these negative values for the header? 

I think you don't need to do this, the `.container` class always center the header section, you don't need to apply any margin-left or right for this.